training-server.yml

```diff
@@ -1,93 +1,156 @@
-# Production tuning of server limits.conf, Postgres and NGINX required, but this will install a minimal OMERO server,
-# OMERO.web with NGINX and systemd configuration controlling both.
+# Install OMERO.server and OMERO.web with a public user on localhost
 
-# Install the latest release, including PostgreSQL on the same server
+- hosts: outreach.openmicroscopy.org
+  pre_tasks:
 
-# users-to-be-added to come from host_vars
+    - name: Install Make Movie script Prerequisite | MEncoder - Repo
+      become: yes
+      yum:
+        name:  http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm
```

Comment:
i remember having issue with that url
Is it more stable now?

Reply:
the playbook was run more than 20 times in last 2 months. Never any issue occured with this url.

Reply:
Also it's not https, do we still want to publicise this?

Reply:
@jburel - I've never noticed a failure at run-time, either. 

@manics - [we already have](https://github.com/openmicroscopy/prod-playbooks/search?utf8=%E2%9C%93&q=http%3A%2F%2Fli.nux.ro%2Fdownload&type=).  If you've got a better suggestion, we're all 👂👂

Reply:
I have not found any repo offering https
getting rid of mencoder is probably the solution
Note that I have not been able to test the movie script via travis

Reply:
Getting rid of mencoder --> the movie script stops to work --> we are going to an outreach with one installed script which is not working

Do I get this right ? Does not seem to me as a good solution.

Reply:
@pwalczysko sorry for the confusion. I meant using a more recent library for encoding
if we want to keep such feature but this will imply that we have to rewrite portion of the script
For nowI think the url used is the best we have found